vochain/apputils.go

```diff
@@ -30,7 +31,7 @@ import (
 // checkProof checks the validity of a census proof (depending on the origin).
 // key is the data to be proof in behalf the censusRoot.
 // In case of weighted proof, this function will return the weight as second parameter.
-func checkProof(proof *models.Proof, censusOrigin models.CensusOrigin, censusRoot, key []byte) (bool, *big.Int, error) {
+func checkProof(proof *models.Proof, censusOrigin models.CensusOrigin, censusRoot, processID, key []byte) (bool, *big.Int, error) {
```

Comment:
Where is this processID used ?

Reply:
This will be used when ECDSA_SALTED signature type is implemented.